### PR TITLE
Add azure postgres flexible server template

### PR DESCRIPTION
**Pull request by fmoor**
[try it out](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fedgedb%2Fedgedb-deploy%2Fazure%2Fazure%2Fedgedb-flexibleserver.json)

### CODE REVIEW

File: azure/edgedb-flexibleserver.json [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    "parameters": {
+        "name": {
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 63,
+            "metadata": {
+                "description": "Unique name for the deployment, used for the Postgres server and EdgeDB container."
+            }
+        },
+        "postgresAdminUsername": {
```
Comment (elprans) [inline azure/edgedb-flexibleserver.json]: There's no good reason to make this configurable.  Let's hardcode to `'edgedb'`

File: azure/edgedb-flexibleserver.json [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 128,
+            "metadata": {
+                "description": "Administrator username for Postgres."
+            }
+        },
+        "postgresAdminPassword": {
```
Comment (elprans) [inline azure/edgedb-flexibleserver.json]: Let's use the same password for both Postgres and EdgeDB to reduce the number of knobs.

File: azure/edgedb-flexibleserver.json [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        "postgresDiskSizeGB": {
+            "type": "int",
+            "minValue": 5,
+            "maxValue": 4194,
+            "defaultValue": 128,
+            "metadata": {
+                "description": "Azure database for PostgreSQL SKU storage size."
+            }
+        },
+        "edgedbAdminUsername": {
```
Comment (elprans) [inline azure/edgedb-flexibleserver.json]: Let's use hardcoded `edgedb` here as well for simplicity.

File: azure/edgedb-flexibleserver.json [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+            }
+        }
+    },
+    "variables": {
+        "api": "2021-06-01",
+        "postgresServerName": "[concat(parameters('name'), '-pg-server')]",
+        "containerGroupName": "[concat(parameters('name'), '-container-group')]",
+        "containerName": "edgedb-server",
+        "containerImage": "edgedb/edgedb:nightly",
+        "backendDSN": "[concat('postgresql://', parameters('postgresAdminUsername'), ':', parameters('postgresAdminPassword'), '@', variables('postgresServerName'), '.postgres.database.azure.com?sslmode=require')]"
```
Comment (elprans) [inline azure/edgedb-flexibleserver.json]: This seems unused?

File: azure/edgedb-flexibleserver.json [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+                "[resourceId('Microsoft.DBforPostgreSQL/flexibleServers', variables('postgresServerName'))]"
+            ]
+        }
+    ],
+    "outputs": {
+        "fqdn": {
+            "value": "[reference(resourceId('Microsoft.ContainerInstance/containerGroups', variables('containerGroupName'))).ipAddress.fqdn]",
+            "type": "string"
+        },
+        "ipaddress": {
```
Comment (elprans) [inline azure/edgedb-flexibleserver.json:187]: let's generate an EdgeDB DSN as output too

Comment (fmoor) [reply azure/edgedb-flexibleserver.json:187]: changed

**Pull request closed without merging**